clients/web/src/lib/api/queries/BannedPlayers.store.ts

```diff
@@ -0,0 +1,120 @@
+import type {OperationResult} from "@urql/core";
+import {gql} from "@urql/core";
+import {LiveQueryStore} from "$lib/api/core/LiveQueryStore";
+
+enum MemberRestrictionType {
+    QUEUE_BAN = "QUEUE_BAN",
+    RATIFICATION_BAN = "RATIFICATION_BAN",
+}
+
+interface MemberProfile {
+    name: string;
+}
+
+interface Member {
+    profile: MemberProfile;
+}
+
+export interface MemberRestrictionEvent {
+    id: number;
+
+    eventType: number;
+
+    message?: string;
+
+    type: MemberRestrictionType;
+
+    expiration: Date;
+
+    reason: string;
+
+    member: Member;
+
+    manualExpiration?: Date;
+
+    manualExpirationReason?: string;
+
+    memberId: number;
+}
+
+export interface BannedPlayersStoreValue {
```

Comment:
Can this be RestrictedPlayersStoreValue? Restriction isn't always a ban, unless this specifically refers to queue bans, in which case it should reference that.